components/brave_today/browser/pref_utils.cc

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_today/browser/pref_utils.h"
+
+#include <string>
+#include <vector>
+
+#include "brave/components/brave_today/browser/brave_news_p3a.h"
+#include "brave/components/brave_today/common/pref_names.h"
+#include "brave/components/l10n/common/locale_util.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+#include "components/prefs/scoped_user_pref_update.h"
+
+namespace brave_news {
+
+void RegisterProfilePrefs(PrefRegistrySimple* registry) {
```

Comment:
Optional nit: maybe these should be inside `brave_news::prefs` or something, it pollutes the `brave_news` namespace a bit.

Reply:
removed the file now